apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -77,261 +79,163 @@ export function InstructorQuestionSettings({
-                            <span>Change QID</span>
-                          </button>
-                        `
-                      : ''}
-                    ${questionGHLink
-                      ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
-                      : ''}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Configuration</th>
-                  <td>
-                    ${resLocals.authz_data.has_course_permission_view
-                      ? html`
-                          <a
-                            href="${resLocals.urlPrefix}/question/${resLocals.question
-                              .id}/file_view/${infoPath}"
-                          >
-                            info.json
-                          </a>
-                          ${resLocals.authz_data.has_course_permission_edit &&
-                          !resLocals.course.example_course
-                            ? html`
-                                <a
-                                  class="btn btn-xs btn-secondary mx-2"
-                                  href="${resLocals.urlPrefix}/question/${resLocals.question
-                                    .id}/file_edit/${infoPath}"
-                                >
-                                  <i class="fa fa-edit"></i>
-                                  <span>Edit</span>
-                                </a>
-                              `
-                            : ''}
-                        `
-                      : ''}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Type</th>
-                  <td>${resLocals.question.type}</td>
-                </tr>
-                <tr>
-                  <th>Topic</th>
-                  <td>
-                    ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
-                      topic: resLocals.topic,
-                    })}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Tags</th>
-                  <td>
-                    ${renderEjs(__filename, "<%- include('../partials/tags') %>", {
-                      tags: resLocals.tags,
-                    })}
-                  </td>
-                </tr>
-                <tr>
-                  <th>Issues</th>
-                  <td>
-                    ${renderEjs(__filename, "<%- include('../partials/issueBadge') %>", {
-                      count: resLocals.open_issue_count,
-                      issueQid: resLocals.question.qid,
-                      suppressLink: resLocals.suppressLink,
-                      urlPrefix: resLocals.urlPrefix,
+            <div class="card-header bg-primary text-white d-flex">Question Settings</div>
+            <div class="card-body">
+              <form>
+                <div class="form-group">
+                  <label for="title" class="h5">Title</label>
```

Comment:
I would be inclined to use normal label styling here (not `h5`). If you want to label this top section as a whole to match the headings on the _other_ sections, you could add a `<h2 class="h5">General</h2>` above this.

Note also that we (I, technically) have sorta/kinda been standardizing on `<h2 class="h4">` on the administrator pages. Ultimately we can go with what we think looks best here, but can we try out `class="h4"` to see how it looks?

Reply:
Added a "General" section to the top and changed to `class="h4" in [381d728](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/381d7287d02a906cb1405663be90d4f84dc38348). How does this look? 
![Screenshot 2024-04-24 at 11 47 59 AM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/960c7aa7-3c47-4eaf-aa64-01756dca8802)

Reply:
LGTM!